src/GEMItem.h

```diff
@@ -76,7 +76,7 @@ class GEMItem {
     GEMItem(const char* const title_, char* linkedVariable_, GEMSelect& select_, void (*saveAction_)());
     GEMItem(const char* const title_, float& linkedVariable_, GEMSelect& select_, void (*saveAction_)());
     GEMItem(const char* const title_, double& linkedVariable_, GEMSelect& select_, void (*saveAction_)());
-    /* 
+     /* 
```

Comment:
Here you just added a space. 

```suggestion
    /* 
```